views/_notes.erb

```diff
@@ -0,0 +1,20 @@
+<script type="text/javascript" src="/javascript/notes.js"></script>
+
+<% unless notes.empty? %>
+  <h3>Notes</h3>
+    <% notes.each do |note| %>
+      <div class="box">
+        <div class="columns is-full">
+          <div class="column">
+            <h4 class="mb-0"><%= note.body %></h4>
+            <small>Created at: <%= note.created_at %></small>
+          </div>
+          <div class="column is-one-fifth has-text-right">
+            <button class="button is-danger is-small delete-note " data-note-id=<%= note.id %>>X</button>
+          </div>
+        </div>
+      </div>
+    <% end %>
+<% end %>
+
+<%= partial(:add_note) %>
```

Comment:
I would just merge `_add_note.erb` directly into `_notes.erb`. I don't think you'd ever want to render `_add_note.erb` separately from `_notes.erb`?

Reply:
You're right.